packages/mobile/src/screens/wallet-screen/components/CoinCard.tsx

```diff
@@ -100,29 +101,48 @@ export const CoinCard = ({ mint, showUserBalance = true }: CoinCardProps) => {
         justifyContent='space-between'
         alignItems='center'
       >
-        <Flex row alignItems='center' gap='l'>
+        <Flex row alignItems='center' gap='l' style={{ flexShrink: 1 }}>
           {isLoading ? <HexagonalSkeleton /> : renderIcon()}
           <Flex column gap='xs'>
             {isLoading ? (
               <CoinCardSkeleton />
             ) : (
               <>
-                <Text variant='heading' size='s'>
-                  {coinData?.name}
-                </Text>
-                <Flex row alignItems='center' gap='xs'>
+                {coinData?.name ? (
```

Comment:
i feel like we don't need this nullcheck. if there's a coin with no name we're gonna be in big trouble.

Reply:
yea we probably dont need to check, but in the event that there is one this will make it look centered

Reply:
ok. then don't need the `?` in the actual text